tests/unit/network_graph/test_graph.py

```diff
@@ -88,6 +88,17 @@ def complex_community_with_trustlines_and_fees_configurable_balances(complex_com
     return complex_community_with_trustlines
 
 
+@pytest.fixture(
+    params=[
+        CurrencyNetworkGraph.find_path_sender_pays_fees,
+        CurrencyNetworkGraph.find_path_receiver_pays_fees
+    ]
+)
+def find_path_sender_then_receiver_pays(community_with_trustlines, request):
```

Comment:
I think this is a bad name for a fixture. The idea of a fixture there is to be transparent in my opinion. So for me the name should be something like `find_transfer_path_function` or so.

Reply:
I don't understand how `find_transfer_path_function` is transparent.
Then I will use this fixture for finding a path not knowing if it will find a path where the sender pays for fees, the receiver pays for fees, or both.
But I can add `_function` to the name of the fixture since it indeed returns a function and does not find a path.
I called it `parametrised_find_transfer_path_function`. Tell me if it is fine for you.

Reply:
> Then I will use this fixture for finding a path not knowing if it will find a path where the sender pays for fees, the receiver pays for fees, or both.

I assumed that is exactly the goal of that fixture. It will give you a function that finds a transfer path, but you dont specify which one. All tests using that should not rely on weather the sender or receiver pays.

Reply:
If you assume that it contains both functions in that order, than I think you should not group them into a single fixture.

Reply:
Example: You its fine to have a list `fruits = [apple, banana, ...]`
but I think it is wrong to have a list `first_apple_then_banana: [apple, banana]`

Reply:
I don't care about the order, I had it named `find_path_sender_and_receiver_pays`, but I thought one might think both the sender and the receiver pay for the fees (like a shared fee).

However, not all tests using `find_transfer_path_receiver_pays_fees` can use `find_transfer_path_sender_pays_fees` (for example if they test the fees calculations), which is why they cannot use this parametrised fixture and I do not want to name it `find_transfer_path_function`.

If the fixture was named `find_transfer_path_function` I would use it in `test_path_with_fees`, which is wrong since the fee returned by the function would not be the same depending on whether the sender or receiver pays for the fee. (see https://github.com/trustlines-network/relay/pull/275/files#diff-1cc3c064efef43ebd1fda8e62895a381R559)